service/v2/db/BingoPostgresAdapter.py

```
@@ -0,0 +1,345 @@
+                        else None,
+                        "service_data = '{}'".format(json.dumps(service_data))
+                        if service_data
+                        else None,
+                    ]
+                    if x is not None
+                ]
+            )
+            cursor.execute(
+                "update indigoservice.library_metadata set {} where library_id = '{}'".format(
+                    assignment, library_id
+                )
```

Ruff (0.14.8)

Possible SQL injection vector through string-based query construction

(S608)